doc/source/aggregation_with_dissolve.rst

```diff
@@ -57,3 +57,14 @@ If we are interested in aggregate populations, however, we can pass different fu
 
 .. toctree::
    :maxdepth: 2
+
+### Dissolve Arguments
```

Comment:
In rst files, the header should in this case be like:

```
Dissolve Arguments
~~~~~~~~~~~~~~~~~~
```

You can see how the other titles are in this file (rst is a bit more complex than markdown .. and this is one of the differences)

Reply:
my mistake! i noticed some differences from markdown and should have done my research PRIOR to submitted this PR. thank you!